### PR TITLE
Made a few tiny changes

### PR DESCRIPTION
**Pull request by DiosAkhilleus**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,21 +39,25 @@ <h1>Examen Apium<img src="beeicon.png" alt="bee logo" class="logo-one"></h1>
             <div class="score-container mobile-visible">
                 <span class="score-label">Score:</span><span class="score-number">0</span>
             </div>
-            <div class="icons"> <i class="fa-solid fa-delete-left" id="delete-button" alt="backspace" title="backspace"></i><i class="fa-solid fa-trash-can" id="clear-button" alt="clear guess" title="clear"></i><i class="fa-solid fa-arrows-rotate" id="refresh-button" alt="refresh page" title="refresh"></i><i class="fa-solid fa-trophy" id="high-scores-button" alt="view high scores" title="high scores"></i><i class="fa-solid fa-circle-info" alt='read rules' title="rules"></i><i class="fa-solid fa-arrow-down-a-z" id="alphabetize-button-mobile" alt="alphabetize successful guesses" title="alphabetize guesses"></i> <i class="fa-solid fa-envelope" id="contact-button" alt="contact" title="contact"></i>  
+            <div class="icons">
```
Comment (DiosAkhilleus) [inline index.html:42]: I just reorganized the HTML here a little bit to make it slightly clearer

File: index.html [hunk 2]
```diff
@@ -39,21 +39,25 @@ <h1>Examen Apium<img src="beeicon.png" alt="bee logo" class="logo-one"></h1>
...
+            <div class="icons"> 
+                <i class="fa-solid fa-delete-left" id="delete-button" alt="backspace" title="backspace"></i>
+                <i class="fa-solid fa-trash-can" id="clear-button" alt="clear guess" title="clear"></i>
+                <i class="fa-solid fa-arrows-rotate" id="refresh-button" alt="refresh page" title="refresh"></i>
+                <i class="fa-solid fa-trophy" id="high-scores-button" alt="view high scores" title="high scores"></i>
+                <i class="fa-solid fa-circle-info" alt='read rules' title="rules"></i>
+                <i class="fa-solid fa-arrow-down-a-z" id="alphabetize-button-mobile" alt="alphabetize successful guesses" title="alphabetize guesses"></i>
+                <i class="fa-solid fa-envelope" id="contact-button" alt="contact" title="contact"></i>  
            </div>
-
```
Comment (DiosAkhilleus) [inline index.html:44]: Removed a few spaces here and there for consistency

File: main.js [hunk 3]
```diff
@@ -6,12 +6,12 @@
 
 class ScoreCard {
   constructor() {
-    this.words = [];
+    this.words = [""];
     this.score = 0;
   }
   addWord(word) {
     if (this.words[0] === "") {
```
Comment (DiosAkhilleus) [inline main.js:13]: You might run into issues here, which is why I slightly modified the instantiation of this.words in the constructor. If you attempt to access this.words[0] when this.words = [] you will likely get an error of some kind. It's possible this is unnecessary if your code handles it in a different way, but it seems safer to me

File: main.js [hunk 4]
```diff
@@ -6,12 +6,12 @@
...
-      this.words[0] = words;
+      this.words[0] = word;
```
Comment (DiosAkhilleus) [inline main.js:14]: Your function parameter is word, so I assume you meant to set this.words[0] to be equivalent to `word` the parameter

File: main.js [hunk 5]
```diff
@@ -55,12 +55,12 @@ const gameDisplay = (() => {
   const contactContainer = document.querySelector(".contact");
 
   const populateBoard = () => {
-    for (i = 0; i < 7; i++) {
+    for (let i = 0; i < 7; i++) {
```
Comment (DiosAkhilleus) [inline main.js:58]: You should instantiate the variable `i` with the keyword `let`

File: style.css [hunk 6]
```diff
@@ -91,14 +91,18 @@ body{
...
     font-size: 1.8rem;
-    width:1px;
-    margin: 0 auto;
+    margin: auto;
+    margin-top: 3px;
+    text-align: center;
+}
+
+.hexagon:hover {
+    cursor: pointer;
```
Comment (DiosAkhilleus) [inline style.css:105]: If you are ever using something on your page as a button, it is easier for the user to recognize it if you change the cursor to the pointer finger on hover

File: style.css [hunk 7]
```diff
@@ -177,6 +181,10 @@ that they form a hexagon */
     display: flex;
 }
 
+.icons:hover {
+ cursor: pointer;
+}
```
Comment (DiosAkhilleus) [inline style.css:186]: Same thing here! Good job though having the icons change color on hover

**Review by DiosAkhilleus — COMMENTED** [review]
Overall, really impressive! I was pleasantly surprised that you used the morphology api I used in my project! Very clean overall with not a whole lot I would change. There were a few tiny things here and there, but I would say that this project is above and beyond what I would've expected at this point in the Odin curriculum! 

I really recommend moving to React as soon as possible! You clearly have a pretty good handle on interacting with JavaScript and HTML, and although there is a bit of a learning curve to React, it ends up being much easier to work with than vanilla JS! 

If you want to make just a front-end project with React, and then move to full-stack, I think that could be a good route to take. It is also certainly feasible to just go straight to full-stack, but I found it easier to learn different technologies incrementally rather than all at once. 

I'll send you some of the resources I found particularly helpful for React, along with a few other resources!

Looking forward to seeing what you make next!

**Comment by sashavining** [comment]
Great! Thank you again for taking a look and leaving such detailed feedback. I'd love to take a look at your React materials!

**Pull request merged**